packages/rest-typings/src/v1/users/UsersUpdateParamsPOST.ts

```diff
@@ -106,6 +107,10 @@ const UsersUpdateParamsPostSchema = {
 					type: 'string',
 					nullable: true,
 				},
+				language: {
+					type: 'string',
```

Comment:
Would it make sense to validate the language instead of accepting any arbitrary value?